upgrade/scripts/upgrade/prepare-assets.sh

```diff
@@ -90,7 +90,9 @@ if [[ -z ${TARBALL_FILE} ]]; then
 
     if [[ -z ${ENDPOINT} ]]; then
         # default endpoint to release.algol60.net
-        ENDPOINT=https://release.algol60.net/csm-1.4/csm/
+        # Example input: CSM_RELEASE=1.5.0-alpha.3
+        # Example output: ENDPOINT=https://release.algol60.net/csm-1.5/csm/
+        ENDPOINT=https://release.algol60.net/csm-$(echo ${CSM_RELEASE} | cut -d'.' -f1,2)/csm/
```

Comment:
this won't work during our testing, we usually put `1.5.0-alpha.3` and we tag doc rpm without alpha.3